src/propertySelector/tpl/label-text.tpl

```diff
@@ -0,0 +1 @@
+<span class="property-description-label">{{{text}}}{{#if alias}}/{{/if}}</span>
```

Comment:
**issue:** The slash separator is also added by the `alias-text` template. It causes double separation in the list. Is it expected? If not, it needs to be fixed.
<img width="197" alt="image" src="https://user-images.githubusercontent.com/1500098/191680673-4f2d13f2-c877-4142-8c6b-66f49162bc53.png">

Reply:
Mockups are designed with double slash, for me also look confusing a bit